spec/ApiMailAdapter.spec.js

```diff
@@ -402,6 +402,49 @@ describe('ApiMailAdapter', () => {
       expect(payload.Message.Body.Text.Data).toBe(examplePayload.text);
       expect(payload.Message.Body.Html.Data).toBe(examplePayload.html);
     });
+
+      describe('convert ZeptoMail API v1.1 payload', () => {
+      it('converts payload for ZeptoMail for single recepient', () => {
+        const payload = converter.zeptomail({api: '1.1',payload: examplePayload});
+        expect(payload.from.address).toEqual(examplePayload.from);
+        expect(payload.to).toBeInstanceOf(Array);    
+        expect(payload.to.length).toBe(1);
+        expect(payload.to[0].email_address.address).toEqual(examplePayload.to);
+        expect(payload.reply_to).toBeInstanceOf(Array);
+        expect(payload.reply_to.length).toBe(1);   
+        expect(payload.reply_to[0].address).toEqual(examplePayload.replyTo);
+        expect(payload.subject).toBe(examplePayload.subject);
+        expect(payload.textbody).toBe(examplePayload.text);
+        expect(payload.htmlbody).toBe(examplePayload.html);
+      });
+
+      it('converts payload for ZeptoMail for multiple recepients', () => {
+        const multipleRecepientExamplePayload = {
```

Comment:
use same syntax as in existing tests for clarity:

```suggestion
        const examplePayload = {
```